rust/kcl-lib/tests/intersect_cubes/artifact_graph_flowchart.snap.md

```diff
@@ -0,0 +1,117 @@
+```mermaid
+flowchart LR
```

Comment:
Not needed in this PR, but thinking we probably need a new artifact type for booleans soon too.

Won't really have any effect now, but once the bools are real (engine finished) then we're going to need to traverse through these operations in the graph for selections.

Reply:
only commenting on this diagram becaus looking at it, you would have no idea that there was a bool in the code.

Reply:
Good call. I added a subtask to #6005.